backgroundJobs/CompoundParser.js

```diff
@@ -8,7 +8,15 @@ const {waitForCpuToGoBelowThreshold} = require("../machineResources")
 const {retry} = require("../utils")
 
 class Compound {
-    constructor(compoundInfo, network, web3, heavyUpdateInterval = 24) {
+    /**
+     * build a compound parser
+     * @param {*} compoundInfo addresses and other informations about the protocol
+     * @param {string} network the name of the network, must be the same as in the indexkey in compoundInfo
+     * @param {Web3} web3 web3 connector
+     * @param {number} heavyUpdateInterval defines the amount of fetch between two heavy updates
+     * @param {number} fetchDelayInHours defines the delay between 2 fetch, in hours
+     */
+    constructor(compoundInfo, network, web3, heavyUpdateInterval = 24, fetchDelayInHours = 1) {
```

Comment:
added fetchDelayInHours parameter, default to 1 and the constructor documentation